src/foam/nanos/controller/ApplicationController.js

```diff
@@ -419,6 +453,34 @@ foam.CLASS({
       });
     },
 
+    function requestCapability(capabilityInfo) {
+      var self = this;
+
+      capabilityInfo.capabilityOptions.forEach((c) => {
+        self.capabilityCache.set(c, false);
+      });
+
+      self.capabilityAquired = false;
+      self.capabilityCancelled = false;
```

Comment:
TODO: I need to check if these variables are still being used; they probably aren't

Reply:
https://nanopay.atlassian.net/browse/NP-1372